Doc/library/tempfile.rst

```diff
@@ -162,7 +162,9 @@ The module defines the following user-callable items:
    by child processes.
 
    Unlike :func:`TemporaryFile`, the user of :func:`mkstemp` is responsible
-   for deleting the temporary file when done with it.
+   for deleting the temporary file when done with it.If a large number of file
```

Comment:
Please add a space after period.
```suggestion
   for deleting the temporary file when done with it. If a large number of file
```

Reply:
Only I am going to comment, using handle tends to confuse, to say so. I think this should be avoided rather than used in Python, again just a comment, I love CPython compilations, I have my own repo, and it is one of the best languages I have come across taking, C and C++ too, but seeing this go in binary mode, is why I decided to go this route rather than interpreter.

Reply:
I have seen many programs in Python with C and/or C++ mixed.

Reply:
And most of this programs are the ones that weigh less, optimized even in threads. The kind of coding that has gone away everything must be fast...